projects/v2/src/app/components/controls/vis-controls.component.html

```diff
@@ -12,7 +12,7 @@
     </mat-panel-description>
   </mat-expansion-panel-header>
 
-  <div class="py-1 pb-4" *ngIf="currentSheet.name === 'all'">
+  <div class="py-1 pb-4" *ngIf="currentSheet.name === 'all' || (currentSheet.name === 'some' && selectedOrgans.length > 8)">
```

Comment:
Why > 8? What's the logic here?

Reply:
Just a condition. if there are more than 13 to 14 the visualisation is rendering very slowly.

Reply:
Ok, regarding slowness it seemed uncharacteristically slow even with just a couple organs selected.